functions/pubsub/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "@google-cloud/pubsub": "^0.28.1",
-    "safe-buffer": "5.2.0"
+    "safe-buffer": "^5.2.0"
```

Comment:
so we don't have to do this here, but I think we've dropped our usage of safe buffer elsewhere.  Do you think this is something we should keep using?  (I honestly don't know).  Also adding @bcoe for thoughts

Reply:
As `Buffer.from` and its like has been available since Node 6, I'd argue it's probably worth dropping.

Reply:
Oh, good call. :sweat_smile: